server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/EmailGetMethod.scala

```diff
@@ -160,9 +161,8 @@ class EmailGetMethod @Inject() (readerFactory: EmailViewReaderFactory,
       .reduceOption(ReadLevel.combine)
       .getOrElse(MetadataReadLevel)
 
-    // TODO make this configurable
-    if (readLevel.equals(FullReadLevel) && ids.size > 5) {
-      SFlux.error(new RuntimeException(s"Too many items in an email read at level FULL. Got ${ids.size} items instead of maximum 5."))
+    if (readLevel.equals(FullReadLevel) && configuration.jmapEmailGetFullMaxSize.isDefined && ids.size > configuration.jmapEmailGetFullMaxSize.get) {
```

Comment:
Extract this out to a method with a meaningful name?

Reply:
is `configuration.jmapEmailGetFullMaxSize.isDefined` necessary here? As you seem to to allocate it with a value by default if it is not defined in the conf?